Makefile

```
@@ -111,10 +111,11 @@ run-integ-test:
 	go test -race -count=1 -timeout 120m -tags=integration ${PACKAGES}
 
 .PHONY: local-integ-test
-local-integ-test: package-custom-resources run-local-integ-test package-custom-resources-clean
+local-integ-test: package-custom-resources custom-resource-tests run-local-integ-test package-custom-resources-clean
```

nit: maybe we should rename this target to just `local-test`? since it's unit + integ tests that don't require network calls
```suggestion
local-test: package-custom-resources custom-resource-tests run-local-integ-test package-custom-resources-clean
```